env.js

```diff
@@ -23,5 +23,6 @@ module.exports = function() {
     syncSentinelDb: notFalse(process.env.SYNC_DB_SENTINEL),
     syncUserMetaDb: notFalse(process.env.SYNC_DB_USER_META),
     syncLogsDb: notFalse(process.env.SYNC_DB_LOGS),
+    syncUsersDb: process.env.SYNC_DB_USERS === 'true',
```

Comment:
Should this still default to off now that security information is removed from the docs?

Reply:
@kennsippell  - now that we cleanse the hashes before inserting to postgres, I'm OK if we default to being on.  I definitely can see the value of having user data side by side with the rest of the lot.  

Thanks for checking!

Reply:
@kennsippell  - now that we cleanse the hashes before inserting to postgres, I'm OK if we default to being on.  I definitely can see the value of having user data side by side with the rest of the lot.  

Thanks for checking!